bentoml/_internal/service/runner.py

```diff
@@ -1,3 +1,35 @@
-class Runner:
-    def __init__(self):
-        pass
+import attr
+
+
+@attr.s
+class _Runner:
```

Comment:
Just `Runner`? since this is already in the `_internal` namespace

Reply:
I forgot about that. I will fix it